src/System.Private.Xml/src/System/Xml/Base64Decoder.cs

```diff
@@ -20,7 +20,7 @@ internal class Base64Decoder : IncrementalReadDecoder
         private int _bits;
         private int _bitsFilled;
 
-        private static readonly string s_charsBase64 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+        private const string CharsBase64 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
```

Comment:
Would it be better to replace this one with the magic ReadOnlySpan?

Reply:
No, the magic only works for bytes, and it's similar in nature to the magic employed for strings.